diff --git a/node_modules/fast-mlkit-translate-text/fast-mlkit-translate-text.podspec b/node_modules/fast-mlkit-translate-text/fast-mlkit-translate-text.podspec
index 9e80876..a845784 100644
--- a/node_modules/fast-mlkit-translate-text/fast-mlkit-translate-text.podspec
+++ b/node_modules/fast-mlkit-translate-text/fast-mlkit-translate-text.podspec
@@ -16,8 +16,8 @@ Pod::Spec.new do |s|
 
   s.source_files = "ios/**/*.{h,m,mm}"
 
-  s.dependency 'GoogleMLKit/Translate', '~> 9.0'
-  s.dependency 'GoogleMLKit/LanguageID', '~> 9.0'
+  s.dependency 'GoogleMLKit/Translate', '>= 8.0.0'
+  s.dependency 'GoogleMLKit/LanguageID', '>= 8.0.0'
 
   # Use install_modules_dependencies helper to install the dependencies if React Native version >=0.71.0.
   # See https://github.com/facebook/react-native/blob/febf6b7f33fdb4904669f99d795eba4c0f95d7bf/scripts/cocoapods/new_architecture.rb#L79.
